### ISSUE TITLE
Update User Roles in Cord

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-api-v3#2472]
Rename the role 'Fundraising' to 'Relationship Manager'

Create a new role called 'Experience Operations' with the permissions in these spreadsheets:
https://docs.google.com/spreadsheets/d/1Y5HPK3M2ECsreDqmR-Z9q7e2sJoL6WzD6low9N_aptc/edit#gid=0
https://docs.google.com/spreadsheets/d/1GBAyYtzztRGgdIJY44XI27obITYCEDkxY_mLk9IS4jE/edit#gid=1963028748

**Comment by atGit2021** [comment]
Hi Seth, just pushed a draft for Bryan to review, but we may need your review/help making sure the permission settings for ExperienceOperations role are correct.  We can test it once it lands in Develop.

**Comment by atGit2021** [comment]
Seth,  If possible, I would like to break out the portion of this issue that pertains to: Rename the role 'Fundraising' to 'Relationship Manager',  into a different issue.  The migration of Fundraising to Relationship Mgr is not as simple as I thought and may have impacts on DOMO if not done correctly, so I think it's better if we move it to a new/separate Github issue and have a separate correlated branch. The currently linked PR just pertains to the new Experience Operations role which is now sitting in review and once approved Carson or Bryan will merge into Develop.
